plugin.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Plugin Name: Meeting Calendar
+ * Description: This provides a way of scheduling recurring meetings, and displaying a calendar or timetable..
+ * Plugin URI: https://github.com/Automattic/meeting-calendar
+ * Author: Automattic
+ * Version: 1.1.0
+ * License: GPL2+
+ * License URI: https://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+function a8c_meeting_calendar_register() {
+
+	// Register our block script with WordPress
+	wp_register_script(
+		'a8c-meeting-calendar',
+		plugins_url('build/index.js', __FILE__),
+		array('wp-blocks', 'wp-element', 'wp-editor')
```

Comment:
Another benefit of using `wp-scripts` is that we don't need to hardcode these dependencies, webpack generates a PHP file with the version & dependencies. Check out [this documentation for how to use it.](https://github.com/WordPress/gutenberg/blob/master/packages/dependency-extraction-webpack-plugin/README.md#wordpress)